### PR TITLE
RBAC: audit log & pretty print returned paths on err

### PR DESCRIPTION
**Pull request by moogacs**
### What's being changed:
this PR 
- adds audit log for all authZ requests 
  e.g.  `action=authorize build_git_commit=66d2875e8 build_go_version=go1.22.4 build_image_tag=feat/rbac build_wv_version=1.28.0-dev request_action=create_roles resources="Role: dddd" results=failed user=jane@doe.com`
  
- pretty print the response on Forbidden 
```
{
    "error": [
        {
            "message": "forbidden: user 'jane@doe.com' has insufficient permissions to create_roles [Role: dddd]"
        }
    ]
}
```
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: usecases/auth/authorization/rbac/manager.go [hunk 1]
```diff
@@ -169,11 +164,70 @@ func (m *manager) Authorize(principal *models.Principal, verb string, resources
...
 		}
 
-		// TODO audit-log ?
+		perm, err := conv.PathToPermission(verb, resource)
+		if err != nil {
+			return err
+		}
+
+		m.logger.WithFields(logrus.Fields{
+			"action":         "authorize",
```
Comment (reyreaud-l) [inline usecases/auth/authorization/rbac/manager.go:173]: Question/suggestion: I'm not familiar with all the other rbac changes. Do all the rbac related log lines share the same action ? This way we can make it "easy" (not really but still) to filter logs and get all rbac related log lines (change, authorization, etc...).

If that's already the case feel free to ignore, but if not maybe for all rbac log adding something to easily filter them would be nice.

Comment (moogacs) [reply usecases/auth/authorization/rbac/manager.go:173]: yes this function is the aggregate for any AuthZ happens

File: usecases/auth/authorization/rbac/manager.go [hunk 2]
```diff
@@ -175,11 +170,70 @@ func (m *manager) Authorize(principal *models.Principal, verb string, resources
 			return err
```
Comment (dirkkul) [inline usecases/auth/authorization/rbac/manager.go:170]: wouldn't we miss any errors here in the log?

Comment (moogacs) [reply usecases/auth/authorization/rbac/manager.go:170]: sorry, don't get it isn't the error logged already ?

**Review by dirkkul — APPROVED** [review]

**Pull request merged**